ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -0,0 +1,193 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Dialogs 1.3
+import "../../../../../imports"
+import "../../../../../shared"
+import "../../../../../shared/status"
+
+ModalPopup {
+    property string commandTitle: "Send"
+    property string finalButtonLabel: "Request address"
+    property var sendChatCommand: function () {}
+
+    id: root
+    title: root.commandTitle
+    height: 504
+
+    property var selectedRecipient
+    onSelectedRecipientChanged: {
+        selectRecipient.selectedRecipient = this.selectedRecipient
+        selectRecipient.readOnly = !!this.selectedRecipient && !!this.selectedRecipient.address
+    }
```

Comment:
What do you think about making the `selectedRecipient` property an alias of the `RecipientSelector` component? And any affected properties can be bound directly to its own properties. This is what I mean haha:
```suggestion
    property alias selectedRecipient: selectRecipient.selectedRecipient
    RecipientSelector {
        id: selectRecipient
        readOnly = !!selectedRecipient && !!selectedRecipient.address
    }
```